### PR TITLE
19713 short form amalgamation outputs

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#19713

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/report-templates/certificateOfAmalgamation.html [hunk 1]
```diff
@@ -24,40 +24,18 @@
...
-                                ({{ entity.identifier }}),
-                                {%else%}
-                                {{ entity.name }}
-                                ({{ entity.identifier }}),
-                                {%endif%}
-                            </div>
-                        </td>
-                        </tr>
-                        {%endif%}
+                        {% if entity != amalgamatingBusinesses[0] %}
```
Comment (PaulGarewal) [inline legal-api/report-templates/certificateOfAmalgamation.html:33]: would this leave us with multiple 'and's ? For ex. would this make amalgamatingbusiness1 **and** amalgamatingbusiness2 **and** amalgamatingbusiness3? I thought we need the 'and' only at the end.

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:33]: image

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:33]: https://preview.uxpin.com/7f4cfe5fe4b0ba15a83d518177efe82a08e99167#/pages/160598905/comment/sitemap

Comment (PaulGarewal) [reply legal-api/report-templates/certificateOfAmalgamation.html:33]: If that's how we're doing it, looks great 👍

**Review by PaulGarewal — APPROVED** [review]

**Pull request merged**